### ISSUE TITLE
Mangadex enabled data-saver (compressed jpegs) for guest accounts

### ISSUE THREAD

**Issue by Darthagnon** [fmd-project-team/FMD#993]
Mangadex has implemented opt-in data-saver function which loads compressed JPEGs for manga. A notice on their website today said "data-saver on by default for guest accounts". Does FMD appear as a "guest" on Mangadex when downloading? This could impact downloaded manga quality. Is there a way to make sure we get the full-quality artworks?

**Comment by Darthagnon** [comment]
Just discovered ```Options >> Websites >> Accounts```. Will try logging in with MangaDex; might be a necessary workaround.

**Comment by Slasar41** [comment]
FMD will appear as a guest on MangaDex website if you're not login within FMD.
You can compare the quality by manually download the images from the website and FMD while login.

**Comment by Darthagnon** [comment]
Provided I'm logged in, in FMD, all should be max quality, right?

**Comment by Slasar41** [comment]
Yes, it's should be max quality like as you logged in on the website.

**Comment by rs3mk** [comment]
You don't seem to need to log into fmd to download in real quality.

**Comment by Darthagnon** [comment]
@rs3mk it was only Mangadex afaik, and a temporary measure. Not sure what you mean by "real quality", as MD said on their homepage if you weren't logged in a few days ago: "we will serve compressed images to guests temporarily".
